docs/contributing/test.md

```diff
@@ -110,19 +114,25 @@ Try this now.
     $ docker run --privileged --rm -ti -v `pwd`:/go/src/github.com/docker/docker dry-run-test /bin/bash
     ```
 
-3.  Run the tests using the `hack/make.sh` script.
+3.  Run the unit tests using the `hack/test/unit` script.
 
     ```bash
-    root@5f8630b873fe:/go/src/github.com/docker/docker# hack/make.sh dynbinary binary cross test-unit test-integration test-docker-py
+    root@5f8630b873fe:/go/src/github.com/docker/docker# hack/test/unit
```

Comment:
I think the commands become a lot more obvious without the prompt

Reply:
I agree that the `root@` `PS1` deducts from readability quite a bit.  It does nicely disambiguate commands out-versus-in-container, but hopefully `$` versus `#` is good-enough for that.

I cleaned the above instance, then 3 others in this file for consistency, and another bunch from `set-up-dev-env.md`.